weaviate/data/crud_data.py

```diff
@@ -7,6 +7,7 @@
 
 from requests.exceptions import ConnectionError as RequestsConnectionError
 
+from weaviate.batch.crud_batch import _check_positive_num
```

Comment:
Should we move this function to `weaviate.util`?

Reply:
yes, done